test/views/payments/ChargeTypeDetailsViewSpec.scala

```diff
@@ -0,0 +1,192 @@
+
+      "have the correct document title" in {
+        document.title shouldBe "Charge Title - Manage your VAT account - GOV.UK"
+      }
+
+      "have the correct page heading" in {
+        elementText(Selectors.pageHeading) shouldBe "Charge Title"
+      }
+
+      "have a period caption" in {
+        elementText(Selectors.caption) shouldBe "1 January 2021 to 31 March 2021"
+      }
+
+      "render breadcrumbs which" should {
+
+        "have the text 'Business tax account'" in {
+          elementText(Selectors.btaBreadcrumb) shouldBe "Business tax account"
+        }
+
+        "link to bta" in {
+          element(Selectors.btaBreadcrumb).attr("href") shouldBe "bta-url"
+        }
+
+        "have the text 'VAT'" in {
+          elementText(Selectors.vatBreadcrumb) shouldBe "Your VAT account"
+        }
+
+        s"link to ${controllers.routes.VatDetailsController.details.url}" in {
+          element(Selectors.vatBreadcrumb).attr("href") shouldBe controllers.routes.VatDetailsController.details.url
+        }
+      }
+
+      "have the correct first column in the first line" in {
+        elementText(Selectors.dueDateKey) shouldBe "Due date"
+      }
+
+      "display the correct due date for the charge" in {
+        elementText(Selectors.dueDateValue) shouldBe "2021-04-08"
+      }
+
+      "have the correct first column in the second line" in {
+        elementText(Selectors.chargeDueKey) shouldBe "Charge due"
+      }
+
+      "display the correct total amount due" in {
+        elementText(Selectors.chargeDueValue) shouldBe "3333.33"
+      }
+
+      "have the correct first column in the third line" in {
+        elementText(Selectors.clearedAmountKey) shouldBe "Amount received"
+      }
+
+      "display the correct cleared amount" in {
+        elementText(Selectors.clearedAmountValue) shouldBe "2222.22"
+      }
+
+      "have the correct first column in the fourth line" in {
+        elementText(Selectors.outstandingAmountKey) shouldBe "Amount left to pay"
+      }
+
+      "display the correct outstanding amount" in {
+        elementText(Selectors.outstandingAmountValue) shouldBe "1111.11"
+      }
+
+      "have a button" which {
+
+        "has the correct button text" in {
+          elementText(Selectors.button) shouldBe "Pay now"
+        }
+
+        "has the correct href location" in {
+          element(Selectors.button).attr("href") shouldBe "unauthenticated-payments-url"
+        }
+      }
+
+      "have a link to the What you owe page" which {
+
+        "has the correct link text" in {
+          elementText(Selectors.whatYouOweLink) shouldBe "Return to what you owe"
+        }
+
+        "has the correct href" in {
+          element(Selectors.whatYouOweLink).attr("href") shouldBe "#"
+          //TODO: add correct link location once this page and the What you owe page have been wired up
+        }
+      }
+    }
+
+    "the user has a cleared amount and no period for the charge" should {
```

Comment:
Can you add in another scenario for when the user only has one period date? As a way to verify your && logic that requires both to render the caption.